src/styles/todo.scss

```diff
@@ -100,3 +109,14 @@
     opacity: 0.5;
   }
 }
+
+.todo__edit-field {
+  width: 100%;
+  height: 100%;
+  font-size: 24px;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.loader__ondelete {
```

Comment:
The classname 'loader__ondelete' should follow the BEM naming convention. Consider renaming it to 'loader__on-delete' or 'loader__delete'.